src/App.css

```diff
@@ -1,3 +1,237 @@
-.app {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@300;500;700;800&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --medium-blue: #307bbe;
+  --pale-grey: #f5f6fa;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --z: #fafafa;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+}
+
+body {
+  background-color: var(--pale-grey);
+  padding: 8% 2% 7% 6%;
+}
+
+.main {
+  margin: 0 2% 4% 0;
+  padding: 0 0 16%;
+  border-radius: 4px;
+  box-shadow: 10px 10px 10px 10px rgba(0, 0, 0, 0.05);
+  background-color: var(--z);
+}
+
+.navigation {
+  margin: 0 0 2%;
+  padding: 1% 6% 1% 6%;
+  background-color: var(--white-two);
+  border: solid 1px var(--white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation-item {
+  display: flex;
+  align-items: center;
+}
+
+.cms {
+  padding-right: 2.938rem;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 2rem;
+  font-weight: bold;
+  color: var(--azure);
+}
+
+.link {
+  text-decoration: none;
+  margin-right: 2rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1.1rem;
+  letter-spacing: 2px;
+  color: #121212;
+}
+
+.active {
+  opacity: 1;
+  color: var(--medium-blue);
+}
+
+.profile {
+  width: 2.813rem;
+  height: 2.813rem;
+  color: var(--azure);
+}
+
+.mybook {
+  margin: 2% 6% 0;
+  padding: 2% 9.188% 1.625% 1.688%;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.title {
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.author {
+  margin: 0.25rem 0 0;
+}
+
+.book-option > button,
+.author {
+  border: none;
+  background-color: var(--white-two);
+  margin-right: 0.938rem;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
```

Comment:
Great job @QueenterJuma 🤝

- [ ] I would recommend you create a single class for this `font-family` rule and call it wherever you wish to use it. This is because you repeated it in various areas which is regarded as a bad practice.